File: app/src/main/java/com/example/amconsofttest/io/realm_models/CompanyRealm.java
package com.example.amconsofttest.io.realm_models;

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;

public class CompanyRealm extends RealmObject {

    @PrimaryKey
    private String name;
    private String catchPhrase;
    private String bs;

    public String getName() {
        return name;
    }

    public CompanyRealm setName(String name) {
        this.name = name;
        return this;
    }

    public String getCatchPhrase() {
        return catchPhrase;
    }

    public CompanyRealm setCatchPhrase(String catchPhrase) {
        this.catchPhrase = catchPhrase;
        return this;
    }

    public String getBs() {
        return bs;
    }

    public CompanyRealm setBs(String bs) {
        this.bs = bs;
        return this;
    }
}

File: app/src/main/java/com/example/amconsofttest/io/realm_models/CompanyRealm.kt
package com.example.amconsofttest.io.realm_models

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey

open class CompanyRealm : RealmObject() {

    @PrimaryKey
    private var name: String? = null
    private var catchPhrase: String? = null
    private var bs: String? = null

    fun getName(): String? {
        return name
    }

    fun setName(name: String): CompanyRealm {
        this.name = name
        return this
    }

    fun getCatchPhrase(): String? {
        return catchPhrase
    }

    fun setCatchPhrase(catchPhrase: String): CompanyRealm {
        this.catchPhrase = catchPhrase
        return this
    }

    fun getBs(): String? {
        return bs
    }

    fun setBs(bs: String): CompanyRealm {
        this.bs = bs
        return this
    }
}
